src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -595,23 +657,16 @@ export class ActivityFillGraphQuerier {
 
     const sampleContributions = [];
     for (let sample = sampleRangeStart; sample < sampleRangeEnd; sample++) {
-      const stackIndex = samples.stack[sample];
-      const sampleCategory =
-        stackIndex !== null
-          ? stackTable.category[stackIndex]
-          : greyCategoryIndex;
-      if (sampleCategory === category) {
```

Comment:
Have you been able to see whether removing the category filtering here brings a performance penalty, or maybe it changes nothing?

Reply:
I think the performance difference is really insignificant. I tried to profile with and without this change and couldn't find any difference.